torch/onnx/utils.py

```diff
@@ -483,14 +496,14 @@ def _model_to_graph(model, args, verbose=False,
                             module=module)
     from torch.onnx.symbolic_helper import _onnx_shape_inference
     if isinstance(model, torch.jit.ScriptModule) or isinstance(model, torch.jit.ScriptFunction):
-        assert example_outputs is not None, "example_outputs must be provided when exporting a ScriptModule or " \
-                                            "ScriptFunction."
-        if isinstance(example_outputs, list):
+        if example_outputs is None:
+            example_outputs = _get_example_outputs(model, args)
+        elif isinstance(example_outputs, list):
+            # example_outpus specified
```

Comment:
@hwangdeyu This branch does not replicate the old behavior, due to missing calling the above snippet

```python
    if isinstance(example_outputs, (torch.Tensor, int, float, bool)):
        example_outputs = (example_outputs,)
```

Reply:
I will try to edit the PR that submits to master, and tag you for review.

Reply:
> I will try to edit the PR that submits to master, and tag you for review.

Sure, that's a necessary modification. Thanks!